clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/IngestionThrottler.java

```diff
@@ -190,14 +206,30 @@ public IngestionThrottler(
                 "non_current_version_aa_wc_leader_records_count",
                 false,
                 EventThrottler.BLOCK_STRATEGY));
+    if (isAdaptiveThrottlerEnabled) {
+      adaptiveIngestionThrottler = new VeniceAdaptiveIngestionThrottler(
+          serverConfig.getAdaptiveThrottlerSignalIdleThreshold(),
+          serverConfig.getNonCurrentVersionNonAAWCLeaderQuotaRecordsPerSecond(),
+          serverConfig.getThrottlerFactorsForNonCurrentVersionAAWCLeader(),
+          serverConfig.getKafkaFetchQuotaTimeWindow(),
+          "non_current_version_non_aa_wc_leader_records_count");
+      adaptiveIngestionThrottler.registerLimiterSignal(adaptiveThrottlerSignalService::isSingleGetLatencySignalActive);
+      adaptiveIngestionThrottler
+          .registerLimiterSignal(adaptiveThrottlerSignalService::isCurrentLeaderMaxHeartbeatLagSignalActive);
+      adaptiveIngestionThrottler
+          .registerLimiterSignal(adaptiveThrottlerSignalService::isCurrentFollowerMaxHeartbeatLagSignalActive);
```

Comment:
Could we double check if this signal computation is ignoring system store? I think system store does not count towards our SLA, but I don't remember if we filter this out during signal computation.
...or maybe it does not matter much if we rollout version-level ingestion reporting everywhere,